### PR TITLE
Remove obsolete UseAPIKey value

### PR DESCRIPTION
**Pull request by imsodin**
See syncthing/syncthing#4438. I didn't add the fswatcher values, as I don't know what how this works. I wanted to simply mimick `rescanIntvs`, but that is missing as well. I would appreciate if someone who knows this stuff could at some point do the necessary modifications.

### CODE REVIEW

File: cmd/ursrv/main.go [hunk 1]
```diff
@@ -245,7 +244,6 @@ func (r *report) FieldPointers() []interface{} {
 		&r.FolderUsesV3.PullOrder, &r.FolderUsesV3.FilesystemType,
 
 		&r.GUIStats.Enabled, &r.GUIStats.UseTLS, &r.GUIStats.UseAuth,
-		&r.GUIStats.UseAPIKey, &r.GUIStats.InsecureAdminAccess,
```
Comment (AudriusButkevicius) [inline cmd/ursrv/main.go]: Removes too much

Comment (imsodin) [reply cmd/ursrv/main.go]: Doh, do I really need to read?...

**Comment by AudriusButkevicius** [comment]
So rescanIntvs seems to be discarded, we should fix that with your addition.
You can use `int[]` in the database, and https://godoc.org/github.com/lib/pq#Int64Array in the struct, which should work.

Alternatively, if you are lazy, you can use JSONB like I did for other things.

**Comment by calmh** [comment]
I was half tempted at some point to change this just to just store the incoming json  blobs directly (in an object db maybe) and then do all parsing later. Maybe that makes more sense when the UR changes more often. Maybe it's not worth the work.

**Comment by AudriusButkevicius** [comment]
The only problem I see is DOS by storing large garbage objects.

**Comment by imsodin** [comment]
May I leave this including the watcher statistics to you? You already know about the system and its requirements, of which I have no clue.

**Comment by AudriusButkevicius** [comment]
This was my first commit to this repo, so I wouldn't say I "know" other than the 300 lines I had to read, unless you are referring to @calmh

**Pull request merged**